dot/system/service.go

```diff
@@ -35,11 +35,6 @@ func (s *Service) SystemName() string {
 	return s.systemInfo.SystemName
 }
 
-// ChainType returns the system's chain type
-func (s *Service) ChainType() string {
-	return s.systemInfo.ChainType
-}
-
```

Comment:
it might make more sense to leave this here, and in `dot` `createSystemService` pass in the database and read it from there and store it in `system.Service`. that was you wouldn't need to pass `storageState` into the rpc system module

Reply:
Yes, this does make more sense, done.